operation.go

```diff
@@ -41,10 +40,12 @@ func (o Operation) MarshalJSON() ([]byte, error) {
 	switch o.Type {
 	case OperationCopy, OperationMove:
 		o.Value = nil
+		o.OldValue = nil
 	case OperationAdd, OperationTest:
 		o.From = emptyPtr
-	case OperationReplace:
-		o.From = pointer(fmt.Sprintf("%v", o.OldValue))
+		o.OldValue = nil
```

Comment:
@joshianiket22 this shouldn't be here should it?  On a replace we WANT to show the oldValue.

Reply:
I've already removed that. Its also not showing up in commit.